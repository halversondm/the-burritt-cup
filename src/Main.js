import React, { Component } from 'react';

class Main extends Component {
    render() {
        return (
            <div className="container">
                <div className="row justify-content-center">
                    <div className="col-lg-8 col-md-10">
                        <div className="post-preview">
                            <h2 className="post-title">
                                Welcome to The Burritt Cup
                    </h2>
                            <h3 className="post-subtitle">
                                What is The Burritt Cup?
                    </h3>
                            <p className="post-meta">The Burritt Cup is a once a year gathering of friends to accomplish the difficult
                                task of golf. The quest for the Cup is accomplished on the third round of golf by playing a scramble
                                format. The teams are chosen at random by the Rules Commissioner. The first, second and fourth
                                rounds of the three-day weekend consist of 18 hole golf each round that determine the other awards
                        of the weekend.</p>
                            <h3 className="post-subtitle">
                                When is The Burritt Cup?
                    </h3>
                            <p className="post-meta">The Burritt Cup is held in June or July every year as decided by the players.</p>
                            <h3 className="post-subtitle">
                                Where is The Burritt Cup?
                    </h3>
                            <p className="post-meta">The Burritt Cup is held in different golf resorts in the Midwest. The players
                                choose the location based on amenities. Locations have included the ritzy Coachman's in Wisconsin
                        and the dumpy Geneva National.</p>
                        </div>
                    </div>
                </div>
            </div>
        );
    }
}

export default Main;