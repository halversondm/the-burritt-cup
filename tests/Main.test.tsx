import Main from '../src/Main';
import {render} from "@testing-library/react";

test("renders without crashing", () => {
  const unit = <Main />;
  const component = render(unit);
  const tree = component.container;
  expect(tree.toString()).toMatchSnapshot();
});
