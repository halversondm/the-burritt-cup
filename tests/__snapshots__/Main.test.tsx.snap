// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders without crashing 1`] = `"<div><div class="container"><div class="row justify-content-center"><div class="col-lg-8 col-md-10"><div class="post-preview"><h2 class="post-title">Welcome to The Burritt Cup</h2><h3 class="post-subtitle">What is The Burritt Cup?</h3><p class="post-meta">The Burritt Cup is a once a year gathering of friends to accomplish the difficult task of golf. The quest for the Cup is accomplished on the third round of golf by playing a scramble format. The teams are chosen at random by the Rules Commissioner. The first, second and fourth rounds of the three-day weekend consist of 18 hole golf each round that determine the other awards of the weekend.</p><h3 class="post-subtitle">When is The Burritt Cup?</h3><p class="post-meta">The Burritt Cup is held in June or July every year as decided by the players.</p><h3 class="post-subtitle">Where is The Burritt Cup?</h3><p class="post-meta">The Burritt Cup is held in different golf resorts in the Midwest. The players choose the location based on amenities. Locations have included the ritzy Coachman's in Wisconsin and the dumpy Geneva National.</p></div></div></div></div></div>"`;
